incubator/etcd/README.md

```diff
@@ -96,78 +97,88 @@ $ kill -9 ETCD_1_PID
 ```
 
 ```shell
-$ kubectl get pods -l "component=${RELEASE-NAME}-etcd"
-NAME                 READY     STATUS        RESTARTS   AGE
-etcd-0               1/1       Running       0          54s
-etcd-2               1/1       Running       0          51s
+$ kubectl get pods -l app=etcd,release=${RELEASE_NAME}
+NAME                READY     STATUS    RESTARTS   AGE
+my-release-etcd-0   1/1       Running   0          3m
+my-release-etcd-2   1/1       Running   0          2m
 ```
 
 After a while:
 
 ```shell
-$ kubectl get pods -l "component=${RELEASE-NAME}-etcd"
-NAME                 READY     STATUS    RESTARTS   AGE
-etcd-0               1/1       Running   0          1m
-etcd-1               1/1       Running   0          20s
-etcd-2               1/1       Running   0          1m
+$ kubectl get pods -l app=etcd,release=${RELEASE_NAME}
+NAME                READY     STATUS    RESTARTS   AGE
+my-release-etcd-0   1/1       Running   0          3m
+my-release-etcd-1   1/1       Running   1          20s
+my-release-etcd-2   1/1       Running   0          2m
 ```
 
-You can check state of re-joining from ``etcd-1``'s logs:
+You can check state of re-joining from ``my-release-etcd-1``'s logs:
 
 ```shell
-$ kubectl logs etcd-1
-Waiting for etcd-0.etcd to come up
-Waiting for etcd-1.etcd to come up
-ping: bad address 'etcd-1.etcd'
-Waiting for etcd-1.etcd to come up
-Waiting for etcd-2.etcd to come up
-Re-joining etcd member
-Updated member with ID 7fd61f3f79d97779 in cluster
-2016-06-20 11:04:14.962169 I | etcdmain: etcd Version: 2.2.5
-2016-06-20 11:04:14.962287 I | etcdmain: Git SHA: bc9ddf2
+$ kubectl logs my-release-etcd-1
++ hostname
++ HOSTNAME=my-release-etcd-1
++ ip r+  get 1
+awk '{print $NF;exit}'
++ IP=172.17.0.8
++ PROTO=https
++ SET_ID=3
++ '[' -e /etcd/member_id ]
++ '[' 3 -ge 1 ]
++ await_host my-release-etcd-1.my-release-etcd.default
++ echo 'Waiting for [my-release-etcd-1.my-release-etcd.default] to come up '
++ true
++ echo -n .
++ ping -W 1 -c 1 my-release-etcd-1.my-release-etcd.default
+Waiting for [my-release-etcd-1.my-release-etcd.default] to come up
+ping: bad address 'my-release-etcd-1.my-release-etcd.default'
++ sleep 1s
++ true
++ echo -n .
++ ping -W 1 -c 1 my-release-etcd-1.my-release-etcd.default
++ break
++ echo -n ' done'
++ add_member
 ...
 ```
 
-## Scaling using kubectl
-
-This is for reference. Scaling should be managed by `helm upgrade`
+## Scaling using Helm
 
-The etcd cluster can be scale up by running ``kubectl patch`` or ``kubectl edit``. For instance,
+Scaling is managed by `helm upgrade`.
 
-```sh
-$ kubectl get pods -l "component=${RELEASE-NAME}-etcd"
-NAME      READY     STATUS    RESTARTS   AGE
-etcd-0    1/1       Running   0          7m
-etcd-1    1/1       Running   0          7m
-etcd-2    1/1       Running   0          6m
+The etcd cluster can be scale up by chaning the number of replicas in the
```

Comment:
Minor typo: `chaning` --> `changing`

Reply:
And "can be scaled up" (scale -> scaled)